sycl/test-e2e/Plugin/interop-level-zero-thread.cpp

```diff
@@ -1,6 +1,6 @@
 // REQUIRES: level_zero, level_zero_dev_kit
-// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out %level_zero_options %threads_lib
-// RUN: %BE_RUN_PLACEHOLDER %t.out
+// RUN: %{build} -o %t.out %level_zero_options %threads_lib
+// RUN: %{run} %t.out
```

Comment:
Did `BE_RUN_PLACEHOLDER` somehow also pipe to FileCheck?    In any case, we'll need to add that now, corect?

Reply:
No, we never ran `FileCheck` on that test. I think the author supposed we need (hence checks below), but enabling that should be a separate PR.